src/goto-symex/symex_target_equation.h

```diff
@@ -208,6 +209,7 @@ class symex_target_equationt : public symex_targett
   bool debug_print;
   bool ssa_trace;
   bool ssa_smt_trace;
+  bool smt_during_symex;
```

Comment:
We declare `bool smt_during_symex;` in `execution_state.h`. Do we need to declare it twice?

Reply:
There is no inheritance between `symex_target_equation.h` and `execution_state.h`, I guess this decl is necessary.